tjdashboard/administrator/sql/install.mysql.utf8.sql

```diff
@@ -27,6 +27,7 @@ ALTER TABLE `#__tj_dashboards`
   ADD KEY `idx_checkout` (`checked_out`),
   ADD KEY `idx_state` (`state`),
   ADD KEY `idx_createdby` (`created_by`),
+  ADD UNIQUE KEY `unqk_alias` (`alias`),

```

Comment:
No need of 2 keys on the same field

Reply:
@coolbung Done, Removed key , Alias need to be unique only